src/common/config_parser.cpp

```diff
@@ -629,8 +629,12 @@ void ConfigParser::addOptionsTranslation(cli::CLIWrapper& cli) {
   addSuboptionsDevices(cli);
   addSuboptionsBatching(cli);
 
-  cli.add<bool>("--optimize",
-      "Optimize speed aggressively sacrificing memory or precision");
+  cli.add<bool>("--optimize16",
+      "Optimize speed aggressively sacrificing memory or precision by using 16bit integer GEMM with intgemm instead of floats. Only available on CPU");
+  cli.add<bool>("--optimize8",
+      "Optimize speed even more aggressively sacrificing memory or precision by using 8bit integer GEMM with intgemm instead of floats. Only available on CPU");
+  cli.add<bool>("--intgemm-shifted",
+      "Use a faster, shifted GEMM implementation. Only available with intgemm8.");
```

Comment:
Does `intgemm8` refer to `--optimize8`? If it applies only to `--optimize8`, could it be collapsed into `--optimize 8bit-shifted` as suggested in my comment above?